src/command-registry.coffee

```diff
@@ -87,6 +88,7 @@ class CommandRegistry
       return disposable
 
     if typeof target is 'string'
+      validateSelector(target)
```

Comment:
@nathansobo you cool validating the selectors up front and throwing errors when they aren't valid? This way we don't need to `try/catch` around any `webkitMatchesSelector` calls